clients/roscpp/include/ros/topic_manager.h

```diff
@@ -210,7 +217,9 @@ class ROSCPP_DECL TopicManager
   bool pubUpdate(const std::string &topic, const std::vector<std::string> &pubs);
 
   void pubUpdateCallback(XmlRpc::XmlRpcValue& params, XmlRpc::XmlRpcValue& result);
+  void pubUpdateCallback(XmlRpc::XmlRpcValue& params, XmlRpc::XmlRpcValue& result, XmlRpc::XmlRpcClientInfo& client_info);
```

Comment:
a general comment, seems like a lot of these parameters can be references to const, i.e. `XmlRpc::XmlRpcValue const &` or  `const XmlRpc::XmlRpcValue &`